sdv/metadata/multi_table.py

```diff
@@ -409,6 +409,8 @@ def update_columns(self, table_name, column_names, **kwargs):
             **kwargs:
                 Any key word arguments that describe metadata for the columns.
         """
+        if not isinstance(column_names, list):
```

Comment:
This wouldn't work for tuples and other list-like objects, something like `pandas.api.types is_list_like` could handle those.

If that's expected then you can leave as is.

Reply:
I think that's expected. Don't imagine a user putting in something more complicated than a list of strings and it's easy enough to convert.